stock-express/src/models/user.model.js

```diff
@@ -77,6 +77,12 @@ const User = {
   getUser: async (id) => {
     return UserModel.findOne({ userId: id });
   },
+  updateHoldings: async(id, holdings) => {
+    return UserModel.findOneAndUpdate({ userId: id }, { holdings: holdings });
+  },
+  updateCash: async(id, amount) => {
+    return UserModel.findOneAndUpdate({ userId: id }, { cash: amount });
+  }
```

Comment:
ini mnding jadiin 1 aja ga buat khusus transaction (buy & sell)? kaya
```js
  transaction: async(id, data) => {
    const { amount, holdings } = data;
    return UserModel.findOneAndUpdate({ userId: id }, { cash: amount, holdings: holdings });
  },
```

Reply:
butuh update history jg btw

Reply:
history next ticket.

terus buat yang itu tadinya gue mau gabung, tapi gue mikir kalo misalnya dia 'beli' cash. kan perlu separate operation most likely. jadi gue pisah just in case

Reply:
hmm ok, tpi kta klo ada 2 call kaya gini buat buy, gw takutnya klo 1 got error (for some reason), cma 1 yng ke update

Reply:
iya gue ngerti sih maksud lu, tapi for now belom ketemu a good solution. prob next meeting

Reply:
okok